DependencyInjection/Configuration.php

```diff
@@ -91,7 +92,7 @@ private function addDbalSection(ArrayNodeDefinition $node) : void
                             ->end()
                             ->children()
                                 ->scalarNode('class')->isRequired()->end()
-                                ->booleanNode('commented')->setDeprecated()->end()
+                                ->booleanNode('commented')->setDeprecated(...$this->getCommentedParamDeprecationMsg())->end()
```

Comment:
if return value of `$this->getCommentedParamDeprecationMsg()` is `[]`, I believe using `...` will cause `ArgumentCountError`

Reply:
fixed - for config versions before `5.1`, a message is provided. Attempted to return `null` for  < 5.1 but that triggered a `TypeError` in an even earlier version before `setDeprecated()` allowed `?string`